apps/frontend/src/app/2_molecules/CollateralRatio/CollateralRatio.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react';
+
+import {
+  Heading,
+  HealthBar,
+  HeadingType,
+  HelperButton,
+  Paragraph,
+} from '@sovryn/ui';
+
+export type CollateralRatioProps = {
+  value?: number;
+  minCRatio: number;
+};
+
+export const CollateralRatio: FC<CollateralRatioProps> = ({
+  value,
+  minCRatio,
+}) => {
+  return (
+    <div className="w-80">
+      <div className="flex items-center justify-between mb-3">
+        <Heading
+          className="text-gray-10 flex items-center"
+          type={HeadingType.h3}
+        >
+          Collateral Ratio
+          <HelperButton className="ml-1.5" content="Collateral Ratio" />
+        </Heading>
+        <Heading
+          className="text-gray-30 flex items-center"
+          type={HeadingType.h3}
+        >
+          {value !== undefined ? `${value}%` : 'N/A'}
+        </Heading>
+      </div>
+      <HealthBar
+        start={90}
+        middleStart={110}
+        middleEnd={150}
+        end={250}
```

Comment:
Story asks healthbar slices to be configurable, i suggest you to have a prop which could accept array of settings like:
```
slices={[
  { from: 0, to: 90, className: 'bg-red' },
  { from: 91, to: 110, className: 'bg-yellow' },
  { from: 110, to: 150, className: 'bg-green' },
  ...
]}
```

Reply:
I think current implementation is fine (although I also like your suggestion), we don't need it to be that flexible at this stage